pkg/handlers/httpproxy/inject.go

```diff
@@ -111,19 +102,8 @@ func (h *httpProxyPatchHandler) GeneratePatches(
 				return err
 			}
 
-			defaultWorkerSelector := clusterv1.PatchSelector{
-				APIVersion: bootstrapv1.GroupVersion.String(),
-				Kind:       "KubeadmConfigTemplate",
-				MatchResources: clusterv1.PatchSelectorMatch{
-					MachineDeploymentClass: &clusterv1.PatchSelectorMatchMachineDeploymentClass{
-						Names: []string{
-							"default-worker",
-						},
-					},
-				},
-			}
-			if err := generatePatch(
-				obj, vars, &holderRef, defaultWorkerSelector, log,
+			if err := patches.Generate(
+				obj, vars, &holderRef, selectors.AllWorkersSelector(), log,
```

Comment:
@mhrabovcin @dkoshkin I changed the http proxy patch selector to target all workers rather than just `default-worker` - do you agree this is correct?